tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -175,7 +175,7 @@ runk-tarball: copy-scripts-for-the-tools-build
 rootfs-nvidia-gpu-image-tarball: agent-tarball busybox-tarball kernel-nvidia-gpu-tarball
 	${MAKE} $@-build
 
-rootfs-nvidia-gpu-initrd-tarball: agent-tarball busybox-tarball kernel-nvidia-gpu-tarball
+rootfs-nvidia-gpu-initrd-tarball: #agent-tarball busybox-tarball kernel-nvidia-gpu-tarball
```

Comment:
Was this change deliberate? Wouldn't this target be useless now?

Reply:
Oh shoot, I need to be more careful, Good catch!!!